go.mod

```diff
@@ -5,15 +5,12 @@ go 1.12
 require (
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
-	github.com/submariner-io/admiral v0.0.0-20190717183806-f2aad3ff7771
-	github.com/submariner-io/submariner v0.0.0-20190708095718-350482d85dd4
+	github.com/submariner-io/admiral v0.0.0-20190910105841-6188b5ef1d22
+	github.com/submariner-io/submariner v0.0.2-0.20190828132721-a11a9a84c90d
 	k8s.io/api v0.0.0-20190222213804-5cb15d344471
 	k8s.io/apimachinery v0.0.0-20190629003722-e20a3a656cff
 	k8s.io/client-go v0.0.0-20190521190702-177766529176
 	k8s.io/klog v0.0.0-20181108234604-8139d8cb77af
 )
 
-replace (
-    github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190223031456-9a5ae4453bd
-	github.com/submariner-io/admiral => github.com/tpantelis/admiral v0.0.0-0.20190715192543-1ada6524e26
-)
+replace github.com/bronze1man/goStrongswanVici => github.com/mangelajo/goStrongswanVici v0.0.0-20190223031456-9a5ae4453bd
```

Comment:
we can remove this now.

Reply:
I'm getting the following error, if I remove goStrongswanVici

go: github.com/bronze1man/goStrongswanVici@v0.0.0-20181105005556-92d3927c899e: parsing go.mod: unexpected module path "github.com/RenaultAI/goStrongswanVici"
go: finding golang.org/x/sys v0.0.0-20190412213103-97732733099d
go: finding golang.org/x/net v0.0.0-20190603091049-60506f45cf65
go: finding golang.org/x/sys v0.0.0-20190606165138-5da285871e9c
go: finding github.com/jpillora/backoff v0.0.0-20170918002102-8eab2debe79d
go: finding golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3
go: finding sigs.k8s.io/yaml v0.0.0-20181102190223-fd68e9863619
go: finding gopkg.in/inf.v0 v0.0.0-20150911125757-3887ee99ecf0
go: error loading module requirements
FATA[0113] exit status 1                                
make: *** [Makefile:11: vendor] Error 1